scripts/cord19/temporal_analysis.py

```diff
@@ -0,0 +1,47 @@
+from pyserini.collection import pycollection
+import pandas as pd
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+    empty_date_docids=[]
+    empty_date_dois=[]
+    empty_date_titles=[]
+
+    date_docids = []
+    date_titles = []
+    date_dois = []
+    dates =[]
+
+    collection = pycollection.Collection('Cord19AbstractCollection', '/Users/stephaniewhoo/URA/pyserini/collections/cord19-2020-05-19')
```

Comment:
should have path hard coded. if it's relative to the download instructions that's fine, even better to have cmd-line arg.